cmd/vic-machine/configure/configure.go

```diff
@@ -252,7 +252,7 @@ func updateSessionEnv(sess *executor.SessionConfig, envName, envValue string) {
 
 func (c *Configure) processCertificates(op trace.Operation, client, public, management data.NetworkConfig) error {
 
-	if !c.certificates.NoTLSverify && (c.certificates.Skey == "" || c.certificates.Scert == "") {
+	if len(c.certificates.Cname) == 0 && !c.certificates.NoTLSverify && (c.certificates.Skey == "" || c.certificates.Scert == "") {
```

Comment:
c.certificates.Cname == ""

Reply:
I thought that for some reason using len on strings was more idiomatic? where did i get that idea?